packages/angular/src/generators/library/files/base/package.json__tpl__

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "<%= npmScope %><%= libFileName %>",
```

Comment:
`npmScope` might not be defined. It's not a required property of the `nx.json` and we're missing the `/` here.
```suggestion
  "name": "<% if (npmScope) { %><%= npmScope %>/<% } %><%= libFileName %>",
```